### DISCUSSION TITLE
Backlog

### DISCUSSION THREAD

**Discussion by alvarix** [alvarix/FLogger#115] [Ideas]
- [ ] Markdown (extended for checklists)
- [ ] auto-save

**Comment by chadcrume** [comment]
Discussion Dec 4

- [ ] Tags as key=value pairs
- [ ] Extend our own markdown to have live application command processing. Type a command to start new entry, dupe current entry, link to another entry, nest another entry
    - [ ] MDX is an example of markdown extended to allow JSX for component structure https://mdxjs.com/
    - [ ] "Dead simple Markdown editor." - https://v2.vuejs.org/v2/examples/index.html?redirect=true
- [ ] Keep a list of FLogger ideas in a shared flog file. Maybe just backend Dropbox shares linked into each own Dropbox App folder for FLogger.
- [x] Add to Chad's list: Looking into changing the Dropbox login to a pop-up
- [x] Add to Alvar's list: App should accept more date formats
- [ ] Line numbers
- [ ] Flog stats 
- [ ] Text Message to flog
- [ ] iOS 2 part gesture 
  - [ ] swipe to go to carousel  index 
  - [ ] Swipe up to close flog
- [ ] auto archive/delete
- [ ] revisions
